src/pages/settings/PaymentsPage.js

```diff
@@ -1,24 +1,109 @@
 import React from 'react';
-import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
-import Navigation from '../../libs/Navigation/Navigation';
+import {Pressable, TextInput, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
 import ROUTES from '../../ROUTES';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Text from '../../components/Text';
 import ScreenWrapper from '../../components/ScreenWrapper';
+import NameValuePair from '../../libs/actions/NameValuePair';
+import {fetch} from '../../libs/actions/User';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    // The PaypalMe Username
+    paypalMeAddress: PropTypes.string,
```

Comment:
Should we rename this to `paypalMeUsername`, which would have it match the UI, and remove the need for the comment on line 17?

Reply:
Yes, that makes more sense. Thanks @flodnv 😄